docker/debian.docker

```diff
@@ -0,0 +1,25 @@
+FROM fleshgrinder/debian:stretch-go1-build
+COPY . /usr/local/src/github.com/trivago/gollum
+WORKDIR /usr/local/src/github.com/trivago/gollum
+RUN make -sj$(nproc) current
+LABEL com.trivago.image="gollum"
+
+# ------------------------------------------------------------------------------
+
+FROM fleshgrinder/debian:stretch
+ENV HEALTHCHECK_PATH=/_ALL_
+ENV HEALTHCHECK_PORT=8080
+ENV LOG_LEVEL=1
+
+COPY --from=0 /usr/local/src/github.com/trivago/gollum/gollum /usr/bin/gollum
```

Comment:
why do a dual-stage build here but not on alpine?

Reply:
There is nothing in the Alpine image that would benefit from a multistage build. You are not compiling Gollum there on your own. I guess you build Gollum on the host to build that image, right? Probably a very bad idea because I know that your system is BSD based (Mac) and not necessarily compatible with Linux. However, I guess the cross-compile functionality of Go makes this less problematic.

Reply:
Well, you cannot cross-compile anymore if you use cgo (which we by default don't).
There are also some dynamic linker problems between linux distributions that are fixed for now but might resurface in the future.
So it does make sense to build on the distribution you want to run on.
The local-build-and-copy mechanism is from a time where we had not yet encountered the problem so it makes sense to fix it. We just did not, yet.